libs/common/src/change-stream/delete-one.test.ts

```diff
@@ -71,37 +59,23 @@ describe('deleteOne', () => {
           auditDetails: generateNoUserLoggedInAuditDetails(),
         });
 
-        expect(mockDeletionsCollection.insertOne).toHaveBeenCalledWith(
-          {
-            collectionName: 'users',
-            deletedDocumentId: documentId,
-            auditRecord: generateMockNoUserLoggedInAuditDatabaseRecord(),
-            expireAt: expect.any(Date) as Date,
-          },
-          { session: mockSession },
-        );
-      });
-
-      it('deletes the requested document', async () => {
-        await deleteOne({
-          db: mockDb,
-          documentId,
+        expect(mockDeletionsCollection.insertOne).toHaveBeenCalledWith({
           collectionName: 'users',
-          auditDetails: generateNoUserLoggedInAuditDetails(),
+          deletedDocumentId: documentId,
+          auditRecord: generateMockNoUserLoggedInAuditDatabaseRecord(),
+          expireAt: expect.any(Date) as Date,
         });
 
-        expect(mockUsersCollection.deleteOne).toHaveBeenCalledWith({ _id: { $eq: documentId } }, { session: mockSession });
-      });
+        it('deletes the requested document', async () => {
+          await deleteOne({
+            db: mockDb,
+            documentId,
+            collectionName: 'users',
+            auditDetails: generateNoUserLoggedInAuditDetails(),
+          });
 
-      it('ends the session', async () => {
-        await deleteOne({
-          db: mockDb,
-          documentId,
-          collectionName: 'users',
-          auditDetails: generateNoUserLoggedInAuditDetails(),
+          expect(mockUsersCollection.deleteOne).toHaveBeenCalledWith({ _id: { $eq: documentId } });
         });
-
-        expect(mockSession.endSession).toHaveBeenCalledTimes(1);
```

Comment:
Is it worth adding a `toHaveBeenCalledTimes` for `mockUsersCollection.deleteOne`?